app/models/openstax/accounts/account.rb

```diff
@@ -37,13 +43,12 @@ def is_anonymous?
     protected
 
     def syncing_or_stubbing
-      OpenStax::Accounts.syncing ||\
-      OpenStax::Accounts.configuration.enable_stubbing?
+      syncing || OpenStax::Accounts.configuration.enable_stubbing?
```

Comment:
The models in this gem automatically send updates to Accounts when they are saved. I need a way to prevent this (when I'm getting updates back from Accounts, so I don't end up in an infinite update loop). That's what the `syncing` attr_accessor does. It used to be a global configuration option, but I realized that turning that on and off was probably bad for multithreading, so I moved it to the Account/Group models.

`enable_stubbing?` on the other hand, is a global configuration that just removes all updates (and it can remain that way, since it's not supposed to be changed while the server is running).